### ISSUE TITLE
Opening multiple files from cli is treated as an argument

### ISSUE THREAD

**Issue by Stoppedpuma** [woelper/oculante#382]
Running oculante example-1.png example-2.png errors instead of opening just those two images. No idea if this should be treated as a bug report or as a feature request.

error: Found argument 'example-2.png' which wasn't expected, or isn't valid in this context

**Comment by woelper** [comment]
This is intended as you can't open more than one image at a time.

**Comment by Stoppedpuma** [comment]
I guess this should be treated as a feature request then, as you state this is "intentional" would you be open to implementing this type of behaviour?

**Comment by woelper** [comment]
I don't think that this would be high priority as a feature request. What would you expect to happen if you pass multiple files on the command line?

**Comment by Stoppedpuma** [comment]
For example, you have multiple files in a directory called `example-x.png` as well as other files which you don't want to open. This is where using oculante `example*` would come into place. Good use cases for this feature are of course are opening all files of a certain extension, opening certain images from different directories, and to match the expectation of how other cli tools work (i.e running cat *.txt to display the contents of all txt files in a directory which also has md files).

**Comment by woelper** [comment]
And what exactly would happen after you open them? Do you mean they would be loaded into a sequence?

**Comment by Stoppedpuma** [comment]
Yes, the user should just be able to go through them with next and previous as if they are the only images laoded.

**Comment by Stoppedpuma** [comment]
0a2e918 makes #385 reproducible again.
